sdk/src/Core/Amazon.Runtime/Pipeline/RetryHandler/RetryHandler.cs

```diff
@@ -90,6 +92,8 @@ public override void InvokeSync(IExecutionContext executionContext)
                     {
                         requestContext.Retries++;
                         requestContext.Metrics.SetCounter(Metric.AttemptCount, requestContext.Retries);
+                        MetricsUtilities.AddMonotonicCounterValue(requestContext, TelemetryConstants.CallAttemptsMetricName, TelemetryConstants.AttemptUnitName, 1);
```

Comment:
Why `1` or what does it represent? I see this in a few places seemingly as a default value.